client/webserver/site/src/css/bootstrap.scss

```diff
@@ -0,0 +1,223 @@
+// Bootstrap
+@import "~bootstrap/scss/functions";
+
+// Locally customized BS variables
+@import "variables.scss";
+
+// Bootstrap modules
+@import "~bootstrap/scss/maps";
+@import "~bootstrap/scss/mixins";
+@import "~bootstrap/scss/root";
+@import "~bootstrap/scss/reboot";
+// @import "~bootstrap/scss/type";
+// @import "~bootstrap/scss/_images.scss";
+// @import "~bootstrap/scss/_code.scss";
+@import "~bootstrap/scss/grid";
+// @import "~bootstrap/scss/tables";
+@import "~bootstrap/scss/forms/labels";
+@import "~bootstrap/scss/forms/form-text";
+@import "~bootstrap/scss/forms/form-control";
+@import "~bootstrap/scss/forms/form-select";
+@import "~bootstrap/scss/forms/form-check";
```

Comment:
Have you looked closely at these to see if we're actually using them. I suspect that the only ones we are **really** using are the `<select>` and `<input type="checkbox">` styles.

Reply:
Yes, these are all used throughout the forms.

Reply:
Yes, `-label`, `-check`, `-control` and `-text` are all used.

Reply:
I commented out the top three and I don't really see a difference.

Reply:
![image](https://github.com/decred/dcrdex/assets/5878500/4dd613e0-1538-4f43-9d26-a5271ea850bf)

![image](https://github.com/decred/dcrdex/assets/5878500/e199c544-98c8-4135-aaa8-cdd6a8bdbebc)